http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/RuntimeServiceInterp.scala

```diff
@@ -665,6 +665,16 @@ class RuntimeServiceInterp[F[_]: Parallel](config: RuntimeServiceConfig,
       context <- ctx.ask
       // should num workers be updated?
       targetNumWorkers <- traverseIfChanged(req.updatedNumberOfWorkers, dataprocConfig.numberOfWorkers)(Async[F].pure)
+      _ <- targetNumWorkers.traverse(_ =>
+        if (runtime.status == RuntimeStatus.Stopped)
+          F.raiseError[Unit](
+            new LeoException(
+              s"${context.traceId.asString} | Bad request. Number of workers can only be updated if the dataproc cluster is Running. Please start your runtime before updating the cluster.",
+              StatusCodes.BadRequest
+            )
+          )
+        else F.unit
+      )
```

Comment:
I thought the original predicate (`runtime.status != RuntimeStatus.Running`) was correct. If we `raiseError` only when `runtime.status == RuntimeStatus.Stopped` then we wouldn't be handling the other non-Running statuses correctly, no? 

I understand that you're trying to make it clearer so if a user's cluster was stopped, they know to start it before attempting to update the cluster but I think the message "Number of workers can only be updated if the dataproc cluster is Running" is sufficient to convey that message.

Reply:
In other words, I suggest having

```scala
        if (runtime.status != RuntimeStatus.Running)
          F.raiseError[Unit](
            new LeoException(
              s"${context.traceId.asString} | Bad request. The Dataproc cluster has status ${runtime.status} " +
              s"whereas number of workers can only be updated if the cluster has status ${RuntimeStatus.Running}.",
              StatusCodes.BadRequest
            )
          )
        else F.unit
```

Reply:
I think the message is helpful, but I'm not gonna fight with you on this if you feel strongly :) will update